### DISCUSSION TITLE
Thank you very much for creating this script tool.

### DISCUSSION THREAD

**Discussion by LinkLee2020** [codecrafting-io/asus-setup-tool#1] [General]
Thank you very much for creating this script tool. It helped me completely uninstall and reinstall Armoury Crate. Prior to using this tool, I had been unsuccessful in uninstalling Armoury Crate using the official uninstallation tool, which resulted in the reinstalled Armoury Crate not recognizing my motherboard.

**Comment by TehnoDisco** [comment]
Big thanks for this tool.

**Comment by codecrafting-io** [comment]
Hi. Glad it worked out for you. Right now I'm having a break, but I will hopefully return at the end of the month with small updates to the project. Thank you.
